doc/before-after-reboot-checks.md

```diff
@@ -0,0 +1,48 @@
+# Before and After Reboot Checks
+
+CLUO provides optional functionality for running arbitrary actions before and
+after the rebooting of a node using node labels and annotations.
+
+## Configuring the `update-operator`
+
+When depolying the `update-operator`, you can provide a comma-separated list of
+node annotations to look for before a reboot should occur, and another one for
+after a reboot has occured. The list is provided as a command line argument to
+the binary -
+
+```bash
+update-operator --before-reboot-annotations anno1,anno2 --after-reboot-annotations anno3,anno4
+```
+
+## Making a check
```

Comment:
My concern was that the vagueness doesn't help the user trying to figure it out. For most purposes, I'd expect people to be writing pods and the description of how its "just a label" and "just an annotation" makes it clear other things could be used?